gulpfile.babel.js

```diff
@@ -148,6 +151,39 @@ gulp.task('mocha', ['clean'], () => {
 // gulp serve for development
 gulp.task('serve', ['clean'], () => runSequence('nodemon'));
 
+const buildImage = (path, spec) => new Promise((resolve, reject) => {
+  const docker = new Docker();
+  docker.buildImage(path, spec, (err, response) => {
+    if (err) {
+      console.log(err);
+    }
+    response.pipe(process.stdout);
+    response.on('error', reject)
+    response.on('end', () => {
+      console.log(`Finished building image: ${spec.t}`);
+      resolve();
+    });
+  });
+});
+
+gulp.task('image-tar', () => {
+  return gulp.src(paths.image)
+    .pipe(tar('gomake-api.tar'))
```

Comment:
does this need to be `path.join(paths.tmp, 'gomake-api.tar')` ? -- otherwise i think it will write the .tar file to the current path